README.rst

```diff
@@ -18,40 +18,40 @@ currently run ML code (e.g. in notebooks, standalone applications or the cloud).
 
-.. |docs| image:: https://img.shields.io/badge/docs-latest-success.svg
+.. |docs| image:: https://img.shields.io/badge/docs-latest-success.svg?style=for-the-badge
     :target: https://mlflow.org/docs/latest/index.html
     :alt: Latest Docs
-.. |labeling| image:: https://github.com/mlflow/mlflow/workflows/Labeling/badge.svg
+.. |labeling| image:: https://img.shields.io/github/workflow/status/mlflow/mlflow/Labeling?label=Labeling&style=for-the-badge&logo=github
     :target: https://github.com/mlflow/mlflow/actions?query=workflow%3ALabeling
     :alt: Labeling Action Status
-.. |examples| image:: https://github.com/mlflow/mlflow/workflows/Examples/badge.svg?event=schedule
+.. |examples| image:: https://img.shields.io/github/workflow/status/mlflow/mlflow/Examples?event=schedule&label=Examples&style=for-the-badge&logo=github
     :target: https://github.com/mlflow/mlflow/actions?query=workflow%3AExamples+event%3Aschedule
     :alt: Examples Action Status
-.. |cross-version-tests| image:: https://github.com/mlflow/mlflow/workflows/Cross%20version%20tests/badge.svg?event=schedule
+.. |cross-version-tests| image:: https://img.shields.io/github/workflow/status/mlflow/mlflow/Cross%20version%20tests?event=schedule&label=Cross%20version%20tests&style=for-the-badge&logo=github
     :target: https://github.com/mlflow/mlflow/actions?query=workflow%3ACross%2Bversion%2Btests+event%3Aschedule
     :alt: Examples Action Status
-.. |pypi| image:: https://img.shields.io/pypi/v/mlflow.svg
+.. |pypi| image:: https://img.shields.io/pypi/v/mlflow.svg?style=for-the-badge&logo=pypi
     :target: https://pypi.org/project/mlflow/
     :alt: Latest Python Release
-.. |conda-forge| image:: https://img.shields.io/conda/vn/conda-forge/mlflow.svg
+.. |conda-forge| image:: https://img.shields.io/conda/vn/conda-forge/mlflow.svg?style=for-the-badge&logo=anaconda
     :target: https://anaconda.org/conda-forge/mlflow
     :alt: Latest Conda Release
-.. |cran| image:: https://img.shields.io/cran/v/mlflow.svg
+.. |cran| image:: https://img.shields.io/cran/v/mlflow.svg?style=for-the-badge&logo=r
     :target: https://cran.r-project.org/package=mlflow
     :alt: Latest CRAN Release
-.. |maven| image:: https://img.shields.io/maven-central/v/org.mlflow/mlflow-parent.svg
+.. |maven| image:: https://img.shields.io/maven-central/v/org.mlflow/mlflow-parent.svg?style=for-the-badge&logo=apache-maven
     :target: https://mvnrepository.com/artifact/org.mlflow
     :alt: Maven Central
-.. |license| image:: https://img.shields.io/badge/license-Apache%202-brightgreen.svg
+.. |license| image:: https://img.shields.io/badge/license-Apache%202-brightgreen.svg?style=for-the-badge&logo=apache
     :target: https://github.com/mlflow/mlflow/blob/master/LICENSE.txt
     :alt: Apache 2 License
-.. |downloads| image:: https://pepy.tech/badge/mlflow
+.. |downloads| image:: https://img.shields.io/pypi/dw/mlflow?style=for-the-badge&logo=pypi
```

Comment:
This used to be inaccurate but it looks like they've fixed it (confirmed parity). Also, downloads per week is slightly more impressive than all-time downloads.